README.md

```diff
@@ -149,18 +155,7 @@ The currently supported configurations are:
 
 ## Supported Boards
 
-### tqma8xqp-1gb
-
-The TQMa8Xx Embedded Module from TQ Group configured with the NXP i.MX8QXP SoC and 1GiB of DDR3 ECC memory.
```

Comment:
Should not delete this information but instead find a better place for it (either in manual or seL4 docs site)